languages/elixir/exercises/concept/binary-matching/.docs/introduction.md

```diff
@@ -0,0 +1,52 @@
+Elixir provides an elegant syntax working with binary data as we have seen with the `<<>>` special form provided for working with _bitstrings_.
+
+## Binary data
+
+The binary type is a specialization on the bitstring type. Where bitstrings could be of any length (any number of [bits][wiki-bit]), binaries are where the number of [bits][wiki-bit] can be evenly divided by 8. That is, when working with binaries, we often think of things in terms of [bytes][wiki-bytes] (8 [bits][wiki-bit]). A [byte][wiki-bytes] can represent integer numbers from `0 - 255`. It is common to work with [bytes][wiki-byte] values in [hexadecimal][wiki-hexadecimal], `0x00 - 0xFF`.
+
+Binaries may use the bitstring special form `<<>>` to define a literal. When defining a literal, we can use integer literals (numbers greater than 255 will overflow and only the last 8 bits of the integer will be used) and string literals. By default, the `::binary` modifier is applied to the value. We can concatenate binaries with the `<>/2` operator.
+
+```elixir
+<<255>> == <<0xFF>>
+<<"Hello">> == <<72, 101, 108, 108, 111>>
+```
+
+A _null-byte_ is another name for `<<0>>`
+
+## Strings are encoded binary data
+
+Why does the bitstring special form `<<>>` allow string literals? It is because string are encoded binaries in Elixir! Strings are encoded in [UTF-8][wiki-utf8] format. That is, they are encoded in 8-bit (one-byte) chunks, which allows them to represent more than 255 characters. Therefore, in elixir, the length of a string may not be the same as its byte representation
+
+```elixir
+# This string has a string length of 5, but is made up of 7 bytes
+string = "hełło"
```

Comment:
This example always annoys me when I see it in the Elixir guides 😁 it uses Polish letters but it's not a valid Polish word. A real translation of "hello" would work great instead - it's "cześć". But then again, it might confuse people who have no idea what it means.

No need to change anything, it's just my "national pride" forcing me to complain 😁

Reply:
😁 Let's fix it! I think it's worthwhile to be culturally aware.